src/stratus-application-tests/src/test/java/stratus/jndi/GeoToolsJndiTest.java

```diff
@@ -66,7 +62,7 @@ public static String postgis_lib_version() {
         return "2.2";
     }
 
-    @Before
+    //@Before
```

Comment:
Any reason for these changes? Does this make it actually work in intellij?

Reply:
nope, they sure don't.  i was just testing and never set it back.  i'll fix it.